datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxClusterResponse.java

```diff
@@ -60,7 +61,7 @@ public SdxClusterResponse(String crn, String name, SdxClusterStatusResponse stat
             SdxClusterShape clusterShape, String cloudStorageBaseLocation,
             FileSystemType cloudStorageFileSystemType, String runtime,
             boolean rangerRazEnabled, Map<String, String> tags, CertExpirationState certExpirationState,
-            String sdxClusterServiceVersion, boolean detached, boolean enableMultiAz) {
+            String sdxClusterServiceVersion, boolean detached, boolean enableMultiAz, String databaseEngineVersion) {
```

Comment:
Nit: Like for its subclass, using a builder here would produce a more readable code.